include/const.hpp

```diff
@@ -106,5 +106,24 @@ static const types::PICFRUPathMap bootFailPIC = {
     {blueridge2s4uIM, systemDbusObj},
     {blueridge1s4uIM, systemDbusObj},
     {fujiIM, everBMCObj}};
+
+static const uint8_t FUNCTION_12 = 12;
+static const uint8_t FUNCTION_13 = 13;
+
+/** PEL SRC data structure
+----------------------------------------------------------------------------
+8 byte header | 4 byte WORD2 |  4 byte WORD3 |  4 byte WORD4 |  4 byte WORD5 |
+4 byte WORD6 |  4 byte WORD7 |  4 byte WORD8 |  4 byte WORD9 | 32 BYTES ASCII
+char string|
+----------------------------------------------------------------------------
+*
+* SRC structure has the following sub sections:
+* 1. 8 byte header  [OFFSET 0 to 7]
+* 2. HEX words each of length 4 bytes [OFFSET 8 to 39]
+* 3. 32 bytes ascii character string [OFFSET 40 to 71]
+*/
+static const uint8_t HEX_DATA_SEC_START_OFFSET = 8;
+static const uint8_t HEX_DATA_SEC_END_OFFSET = 39;
```

Comment:
why 39 here ?

Reply:
i modified to take last hex word start offset and not end offset